src/test/java/edu/kit/compiler/transform/TransformContextTest.java

```diff
@@ -50,7 +45,7 @@ private MethodNode initMethod(String input) {
         this.stringTable = lexer.getStringTable();
         this.typeMapper = new TypeMapper(namespaceMapper, lexer.getStringTable());
         for (ClassNode classNode: node.getClasses()) {
-            ClassEntry entry = typeMapper.getClassEntry(classNode);
+            typeMapper.getClassEntry(classNode);
```

Comment:
Is this call necessary?

Reply:
@N-Maas

Reply:
No, I think this is a pure function anyway